src/components/PositioningPortal.story.tsx

```diff
@@ -27,6 +27,7 @@ export default {
           onOpen: () => updateArgs({ isOpen: true }),
           onClose: () => updateArgs({ isOpen: false }),
           onShouldClose: () => updateArgs({ isOpen: false }),
+          closeOnScroll: false,
```

Comment:
This prop should not be set in the decorator.
I would rather leave it in its default value in storybook.

Reply:
yes sure